src/mudlet.cpp

```diff
@@ -2353,10 +2370,26 @@ void mudlet::slot_irc()
 }
 
 void mudlet::slot_discord()
+{
+    Host* pHost = getActiveHost();
+    QString invite;
+    if (pHost) {
+        invite = pHost->getDiscordInviteURL();
+    }
+    openWebPage(invite.isEmpty() ? mMudletDiscordInvite : invite);
+}
+
+void mudlet::slot_mudlet_discord()
 {
     openWebPage(mMudletDiscordInvite);
 }
 
+void mudlet::toggleMudletDiscordVisible(bool vis){
```

Comment:
Our coding style means this should be:
```suggestion
void mudlet::toggleMudletDiscordVisible(bool vis)
{
```